packages/components/radio/src/radio-group.tsx

```diff
@@ -131,7 +143,12 @@ export const useRadioGroup = <
 
   const getRadioProps: PropGetter<
     { value?: Y },
-    { checked?: boolean; isChecked?: boolean; value?: Y }
+    {
+      checked?: boolean
+      /** @deprecated Use `checked` instead */
```

Comment:
I don't think it's necessary.

Reply:
Fixed.
https://github.com/yamada-ui/yamada-ui/pull/3548/commits/15f131b106a54656c8ab6d02800e1dedbcdffcc6#diff-4dac08b3f94fe1468c3ff0798dcf17523f6569ff0865ff9ff05326885e55210dL148